src/rpc/mining.cpp

```diff
@@ -361,13 +361,12 @@ UniValue prioritisetransaction(const UniValue& params, bool fHelp)
 {
     if (fHelp || params.size() != 3)
         throw runtime_error(
-            "prioritisetransaction <txid> <priority delta> <fee delta>\n"
+            "prioritisetransaction <txid> <fee delta>\n"
             "Accepts the transaction into mined blocks at a higher (or lower) priority\n"
             "\nArguments:\n"
             "1. \"txid\"       (string, required) The transaction id.\n"
-            "2. priority delta (numeric, required) The priority to add or subtract.\n"
-            "                  The transaction selection algorithm considers the tx as it would have a higher priority.\n"
-            "                  (priority of a transaction is calculated: coinage * value_in_satoshis / txsize) \n"
+            "2. priority delta (numeric, deprecated) We no longer have the concept of priority.\n"
+            "                  This should be set to zero.\n"
```

Comment:
The usage line and arguments need to be consistent. Either have an explicit zero in the usage line, or remove the priority delta completely.

Additionally, as is, this API now ignores the priority setting, so mining pools who aren't aware of this change might just be setting priority delta and not realise that it now does nothing. We should either enforce that the priority delta is zero, or remove it.

I'm on the fence about which of these to do. Upstream chose to completely remove the field and break backwards compatibility, but they did so three years ago, and mining pools will have long-since updated. Thus two-argument usage is not unreasonable for our mining pools to upgrade to. OTOH, if the mining pools were only setting the priority delta, removing that would cause their current priority delta to be interpreted as a fee delta. I'd prefer a visible change that will result in an RPC error: either (if left) forcing it to be zero, or (if removed) forcing there to be only two arguments.

Reply:
i'm partial to removing the priority delta argument and forcing there to be only two arguments.